src/redisearch_rs/inverted_index_bencher/src/ffi.rs

```diff
@@ -154,4 +182,45 @@ mod tests {
             );
         }
     }
+
+    #[test]
+    fn test_encode_freqs_only() {
+        // Test cases for the frequencies only encoder and decoder. These cases can be moved to the Rust
+        // implementation tests verbatim.
+        let tests = [
+            // (frequency, delta, expected encoding)
+            (0, 0, vec![0, 0, 0]),
+            (0, 1, vec![0, 1, 0]),
```

Comment:
Not sure if it matters, but I don't think `frequency 0` is possible (why do we add this document to this inverted index if it doesn't have this term?)